examples/aws/jaeger-hotrod-on-ec2/setup-jaeger-hotrod.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Script to install and run the Jaeger Hot R.O.D. sample application.
+# Installs docker, docker-compose, writes configuration files, then runs 'docker-compose up'
```

Comment:
Do we want to provide details as to where it installs all of the components? (not that they couldn't read the code script and find it...)

Reply:
Added in-line comments to make the script a bit more readable. Also added a README to the directory which links to the various components used here.